docs/api/1.x/accounts.rst

```diff
@@ -294,3 +294,166 @@ Or delete some account:
 ::
 
     $ http DELETE http://localhost:8888/v1/accounts/sam-body --auth admin:s3cr3t
+
+
+
+.. _accounts-validate:
+
+Validate accounts
+=================
+
+If the ``account validation`` option in :ref:`the settings
+<settings-accounts>` has been enabled, account IDs need to be valid email
+addresses: they need to match the regexp in the
+``account_validation.email_regexp`` setting.
+
+.. sourcecode:: bash
+
+    $ echo '{"data": {"id": "bob@example.com", "password": "azerty123"}}' | http POST http://localhost:8888/v1/accounts --verbose
+
+If the user was created, an email will be sent to the user with a link to a
```

Comment:
Isn't it already linked a few lines above, at the top of the section ? (line 306 in this diff)

Reply:
There's a link to the settings section a few lines above (line 306 in this div). In the settings section, I moved the "email configuration" closer to the beginning of the section instead of the very bottom of it.